packages/notification-services-controller/src/NotificationServicesController/NotificationServicesController.ts

```diff
@@ -210,12 +210,28 @@ export type AllowedActions =
   | NotificationServicesPushControllerUpdateTriggerPushNotifications;
 
 // Events
-export type NotificationServicesControllerMessengerEvents =
+export type NotificationServicesControllerChangeEvent =
```

Comment:
nit: Following the naming convention for controllers allows consumers to be certain of what this type does without having to go digging into its implementation.

```suggestion
export type NotificationServicesControllerStateChangeEvent =
```

Reply:
Noted! I will add these changes in the next set of core PRs we have 👍🏾